src/services/otp.ts

```diff
@@ -28,7 +29,7 @@ export default class OtpService {
             throw new AccessDeniedError('Exceeded otps limits per day', undefined, ProcessCode.OtpSendAttemptsExceeded)
         }
 
-        const value: number = Math.floor(1000 + Math.random() * 9000)
+        const value: string = this.generateOtp(4)
 
         await this.notificationService.sendSms(phoneNumber, SmsTemplateCode.Otp, user, headers, value.toString())
```

Comment:
>  value.toString()

value has already become a string https://github.com/diia-open-source/be-user-service/pull/99/commits/c056296d35b3634993973321afcd7e63939c2dcf#diff-00fedd47698c4954f50bd0b9a727028195d4eb26dfaba6d07dbf153d870b142aR32